upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```diff
@@ -698,10 +737,42 @@ spec:
           initialDelaySeconds: 60
           periodSeconds: 10
           timeoutSeconds: 3
+{{- if .EtcdManaged }}
+        volumeMounts:
+        - mountPath: /var/lib/etcd-config
+          name: etcd-config-path
+          readOnly: true
+        - mountPath: /var/lib/etcd-secrets
+          name: etcd-secrets
+          readOnly: true
+{{- end }}
       hostNetwork: true
+{{- if .EtcdManaged }}
+      # In managed etcd mode, Cilium must be able to resolve the DNS name of
+      # the etcd service
```

Comment:
Did you forget to set the `dnsPolicy` here?

Reply:
No, I removed that part on purpose. Changing the DNS policy breaks cilium because it can no longer look up the internal cluster DNS (since we use the public DNS entry for that). The change on line 14 (` "etcd.operator": "true"`) removes the need to change the DNS policy.
That particular change hasn't made it into the cilium helm templates yet, if you are comparing to what that one outputs.

Reply:
You have a conditional wrapped around nothing but comments here. Perhaps remove the conditional and comments as well?

Reply:
A thanks. I was certain I removed that one. :man_facepalming: